### DISCUSSION TITLE
need hexagon force real quick

### DISCUSSION THREAD

**Discussion by zephyrside** [tfdsoft/famidash#176] [Q&A]
hey @nicco1690 can you send the version of the master module with hexagon force in it
it's not present in the current version so imma merge it in

**Comment by aquamarine-axo** [comment]
unfortunately I no longer have the module because that was when I was running Windows on my laptop and I've since had to wipe both drives in it multiple times to install Linux, and the module file was stored on the drive that was too big to get backed up so I don't have it anymore sadly
